.mvn/settings.xml

```diff
@@ -16,6 +16,13 @@
         <server>
             <id>gh-pages</id>
         </server>
+        <server>
+            <id>gh-pages-scm</id>
```

Comment:
Can you please add some notes to the `Releasing` doc about these necessary `settings.xml` entries?

Reply:
sure - logging off to watch rest of the game but will take a further look early in the week.

Reply:
> sure - logging off to watch rest of the game but will take a further look early in the week.

I notice you didn't say *which* week. 😁 

Happy to finish this for you if you can just quickly summarize in a reply why the differently-named gh_pages config.   And I'll probably remove the dependencies that I don't think are needed.

Reply:
sorry saw this earlier today and realized I forgot about it.  this can certainly be cleaned up and I know how now to address scm publish.  In fact, you probably could find that if you look at maven-parent github project.  That was updated to not use the site deploy during release but rather publish and configured to use it directly from the site already built rather than checkout which was causing this to be slow.  I was going to look more at that this coming weekend to see how the scm works (wrote it down so I don't forget) but if you have time, try seeing what they did that might finish this off.  See here https://github.com/apache/maven-parent/blob/master/maven-skins/pom.xml

Reply:
I might take a peek but I'm pretty slammed this week/month as well.  If this is still open in April I'll deal with it. :)

Reply:
OK, peeked. Config looks pretty simple.  It's a matter of testing and documenting the appropriate mvn commands (apparently no longer staging) to deploy manually.  All of which seems more complicated than just pushing to the site branch upstream.  And I don't know what mvn commands are needed to make that work... what we have now works and I'd be making work for myself breaking it.

So, yeah.  Leaving this for you to work, probably after you've got it working elsewhere.

Reply:
I think provided what I'm doing works out, the 'gh-pages-scm' will be removed later entirely.  site-deploy will be disabled so that won't be used.  The scm publish intead uses 'gh-pages' directly as a branch from the pom.